contracts/FractalUsul.sol

```diff
@@ -0,0 +1,74 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+// Usul was previously named "Seele" and SekerDAO was TokenWalk
+// that's where this naming differences are coming from
+import "@tokenwalk/seele/contracts/Usul.sol";
+
+abstract contract FractalUsul is Usul {
+  struct Transaction {
+    address to;
+    uint256 value;
+    bytes data;
+    Enum.Operation operation;
+  }
+  
+  struct Proposal {
+    bool canceled;
+    uint256 timeLockPeriod; // queue period for safety
+    bytes32[] txHashes;
+    Transaction[] transactions;
+    uint256 executionCounter;
+    address strategy; // the module that is allowed to vote on this
+  }
+
+  event ProposalMetadataCreated(
+    uint256 proposalId, 
+    string title, 
+    string description, 
+    string documentationUrl
+  );
+
+  // mapping of proposal id to proposal
+  mapping(uint256 => Proposal) public override proposals;
```

Comment:
This variable can't be overriden, but also doesn't need to be, since we can access the mapping declared in `Usul.sol`